src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/store/collector/NodeConfigCollectorTest.java

```diff
@@ -53,12 +56,14 @@ public void init() {
     cacheMaxSize = new Cache_Max_Size(5);
     heapMax = new Heap_Max(5);
     nodeConfigCollector = new NodeConfigCollector(1, threadPool_QueueCapacity, cacheMaxSize, heapMax);
+    nodeKey = new NodeKey(new InstanceDetails.Id("node1"), new InstanceDetails.Ip("127.0.0.0"));
```

Comment:
let's read node key from getInstanceDetails instead of hardcoding it

Reply:
Done